crowbar_framework/app/views/barclamp/monasca/_edit_attributes.html.haml

```diff
@@ -6,19 +6,32 @@
 
     %fieldset
       %legend
-        = t(".master_header")
-
-      = string_field %w(master cmm_tarball_url)
-      = password_field %w(master influxdb_mon_api_password)
-      = password_field %w(master influxdb_mon_persister_password)
-      = password_field %w(master database_notification_password)
-      = password_field %w(master database_monapi_password)
-      = password_field %w(master database_thresh_password)
-      = password_field %w(master database_logapi_password)
-      = password_field %w(master keystone_cmm_operator_user_password)
-      = password_field %w(master keystone_cmm_agent_password)
-      = password_field %w(master keystone_admin_agent_password)
-      = password_field %w(master database_grafana_password)
+        = t(".metric_agent_header")
+
+      = boolean_field %w(metric_agent system_only)
+      = string_field %w(metric_agent log_dir)
+      = select_field %w(metric_agent log_level), :collection => :metric_agent_log_levels
+      = integer_field %w(metric_agent statsd_port)
```

Comment:
@dirkmueller I think user should have possibility to configure statsd port for monasca-agent. We had a problem on Openstack installation where other service is using this port (for example this was a gnocchi server which as default also uses 8125 port as default). If port is already used agent won't start. So I think user should be able to configure this before installation.